jalezi/py-cepimose
# FIX dashboards queries (#71)

* fix lab dashboard queries

* add vaccinations new query model

* add third_dose property to VaccinationByDayRow

* fix _parse_vaccinations_by_day() and test

* fix data vaccination_supplied_by_manufacturer

* test vaccination_supplied_by_manufacturer

update assertion data row

* fix data vaccinations_by_region_by_day()

change parser to _parse_vaccinations_by_day

* test vaccinations_by_region_by_day adjust ...

...expectations

* add new R values: 2, 6

_parse_vaccinations_by_day()

* fix _get_default_by_age_group_command()

add new query

* use _parse_vaccinations_by_day in...

... vaccinations_by_age_group()

* !test vaccinations_by_age_group adjust assertions

we need more test for certain age groups where R == 2 || 6

R == 2
AgeGroup.GROUP_75_79 on 2021-10-10
AgeGroup.GROUP_80_84 on 2021-10-10

* fix manufacturer used query

* fix manufacturer used parser due to new query

see: c1a068e7

* test fix manufacrurer used assertions

* test skip not importatant tests

* vaccinations_by_manufacturer_used: added test for timestamps

* fix absurdly high numbers (eg leaked timestamps)

* vaccinations_by_manufacturer_used: added test for

2021-01-29: R=30
2021-10-13: R=28

* test: lab test sanity checks

* fix lab PCR and HAT today queries

Co-authored-by: Štefan Baebler <stefan.baebler@gmail.com>
